driver/razerkbd_driver.c

```diff
@@ -565,18 +565,18 @@ static ssize_t razer_attr_write_keyswitch_optimization(struct device *dev, struc
     // Toggle Keyswitch Optimization
     switch(usb_dev->descriptor.idProduct) {
     case USB_DEVICE_ID_RAZER_HUNTSMAN_V2:
-        report = razer_chroma_misc_set_keyswitch_optimization(NOSTORE, mode);
+        report = razer_chroma_misc_set_keyswitch_optimization_command1(mode);
         report.transaction_id.id = 0x1f;
         razer_send_payload(usb_dev, &report);
-        report = razer_chroma_misc_set_keyswitch_optimization(VARSTORE, mode);
+        report = razer_chroma_misc_set_keyswitch_optimization_command2(mode);
         report.transaction_id.id = 0x1f;
         razer_send_payload(usb_dev, &report);
         break;
     default:
-        report = razer_chroma_misc_set_keyswitch_optimization(NOSTORE, mode);
+        report = razer_chroma_misc_set_keyswitch_optimization_command1(mode);
         report.transaction_id.id = 0x1f;
         razer_send_payload(usb_dev, &report);
-        report = razer_chroma_misc_set_keyswitch_optimization(VARSTORE, mode);
+        report = razer_chroma_misc_set_keyswitch_optimization_command2(mode);
         report.transaction_id.id = 0x1f;
         razer_send_payload(usb_dev, &report);
     }
```

Comment:
Looking at this again, I was questioning the need for this switch case at all at this point. This is the only device making use of this feature at the moment. There are other Huntsman V2 keyboards that state in their manuals they support key optimization. It seems likely they might use the exact same commands. I can remove the switch block if you want.

Reply:
generally it's ok, just see other comment I made here